core/domain/email_manager.py

```diff
@@ -164,5 +181,60 @@ def send_post_signup_email(user_id):
         EMAIL_FOOTER.value)
 
     _send_email(
-        user_id, feconf.SYSTEM_COMMITTER_ID, email_models.INTENT_SIGNUP,
+        user_id, feconf.SYSTEM_COMMITTER_ID, feconf.EMAIL_INTENT_SIGNUP,
         email_subject, email_body)
+
+
+def require_valid_intent(intent):
+    if intent not in feconf.VALID_MODERATOR_ACTIONS:
+        raise Exception('Unrecognized email intent: %s' % intent)
+
+
+def _get_email_config(intent):
+    require_valid_intent(intent)
+    return config_domain.Registry.get_config_property(
```

Comment:
That doesn't seem like a message you could actually send to a user. Couldn't you make it some generic message about what has happened so that editing it is optional?

Also could you move this variable to feconf? Then all the email-related texts will be in the same place, and people are more likely to notice and edit them.